SUMMARY.md

```diff
@@ -16,12 +16,12 @@
 * [Delivery Recipe](delivery_recipe/README.md)
     * [Risk Management](delivery_recipe/risk_management.md)
         * [Positive Risk Attitude](delivery_recipe/positive_risk_attitude.md)
-    * User Stories
```

Comment:
Agreed. I'll update the nav. This will fit better when I've written my pages on Kanban